cmd/utils/flags.go

```diff
@@ -1877,6 +1889,13 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		cfg.Genesis = core.DefaultOdysseyGenesisBlock()
 		SetDNSDiscoveryDefaults(cfg, params.OdysseyGenesisHash)
 		cfg.Miner.GasPrice = big.NewInt(params.GWei * 16)
+	case ctx.Bool(StoryFlag.Name):
+		if !ctx.IsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 1415
+		}
+		cfg.Genesis = core.DefaultStoryGenesisBlock()
+		SetDNSDiscoveryDefaults(cfg, params.StoryGenesisHash)
+		cfg.Miner.GasPrice = big.NewInt(params.GWei * 16)
```

Comment:
This was a temporary fix to bake in gasprice setting only, other settings are [all default ones](https://github.com/piplabs/story-geth/blob/main/eth/ethconfig/config.go#L50). Do we want to customize other settings as well?

If we want to customize other settings, we probably want to create another config in parallel with the [default one](https://github.com/piplabs/story-geth/blob/main/eth/ethconfig/config.go#L50) called `StoryConfig` or something?

Reply:
Yes we can, but I think this would be ok. wdyt @edisonz0718 ?

Reply:
Our [geth.toml](https://github.com/storyprotocol/network-launcher/blob/main/config/odyssey-devnet/validator1-binary/geth/geth.toml) for devnet has quite some diff from the default settings. We may want to review and see if there are additional customizations needed.

Reply:
Yes, gas price is equal but some others are not equal. I'm good with merging it now, but we should remember to go over the settings again and see if the default ones are good.